src/DataService/Player/Player.php

```diff
@@ -0,0 +1,2460 @@
+     * @var array<string, Integer>
+     */
+    protected $regionWidths;
+
+    /**
+     * URL parameters to add to Public URL requests.
+     *
+     * @var string
+     */
+    protected $releaseUrlParams;
+
+    /**
+     * The identifier of the restriction to apply to this player.
+     *
+     * @var \Psr\Http\Message\UriInterface
+     */
+    protected $restrictionId;
+
+    /**
+     * Indicates whether to display the air date in the release list component.
+     *
+     * @var bool
+     */
+    protected $showAirdate;
+
+    /**
+     * Indicates whether to include the All option in the category list component.
+     *
+     * @var bool
+     */
+    protected $showAllChoice;
+
+    /**
+     * Indicates whether to display the author in the release list and clip info components.
+     *
+     * @var bool
+     */
+    protected $showAuthor;
+
+    /**
+     * Indicates whether to display the media bitrate in the release list component.
+     *
+     * @var bool
+     */
+    protected $showBitrate;
+
+    /**
+     * Indicates whether to show the full video time in the player control rack.
+     *
+     * @var bool
+     */
+    protected $showFullTime;
+
+    /**
+     * Indicates whether to show the Most Popular option in the category list component.
+     *
+     * @var bool
+     */
+    protected $showMostPopularChoice;
+
+    /**
+     * Indicates whether to display the previous and next buttons in the player control rack.
+     *
+     * @var bool
+     */
+    protected $showNav;
+
+    /**
+     * Indicates whether to randomize the contents of the release list.
+     *
+     * @var bool
+     */
+    protected $shuffle;
+
+    /**
+     * Identifier for the skin object to apply this player.
+     *
+     * @var \Psr\Http\Message\UriInterface
+     */
+    protected $skinId;
+
+    /**
+     * The height of the thumbnail images in the release list.
+     *
+     * @var int
+     */
+    protected $thumbnailHeight;
+
+    /**
+     * The width of the thumbnail images in the release list.
+     *
+     * @var int
+     */
+    protected $thumbnailWidth;
+
+    /**
+     * The name of this object.
+     *
+     * @var string
+     */
+    protected $title;
+
+    /**
+     * The date and time this object was last modified.
+     *
+     * @var \DateTime
+     */
+    protected $updated;
+
+    /**
+     * The id of the user that last modified this object.
+     *
+     * @var \Psr\Http\Message\UriInterface
+     */
+    protected $updatedByUserId;
+
+    /**
+     * Indicates whether to float the control rack over the media area of the player.
+     *
+     * @var bool
+     */
+    protected $useFloatingControls;
+
+    /**
+     * This object's modification version, used for optimistic locking.
+     *
+     * @var int
+     */
+    protected $version;
+
+    /**
+     * The width of the player.
+     *
+     * @var int
+     */
+    protected $width;
+
+    /**
+     * Returns The date and time that this object was created.
```

Comment:
Returns **the**

Reply:
/me goes to do another search / replace!